app/liquid_tags/asciinema_tag.rb

```diff
@@ -32,3 +32,5 @@ def validate(id)
 end
 
 Liquid::Template.register_tag("asciinema", AsciinemaTag)
+
+UnifiedEmbed.register(AsciinemaTag, regexp: %r{https://asciinema.org/a/(?<id>\d+)})
```

Comment:
Reading all of these: `UnifiedEmbed.register(KLASS, regexp: SOMETHING)` I really like the idea of having a uniform constant name that's scoped to the `KLASS`.

```ruby
class AsciinemaTag
  REGISTRY_REGEXP = %r{https://asciinema.org/a/(?<id>\d+)}
end

UnifiedEmbed.register(AsciinemaTag, regexp: AsciinemaTag:: REGISTRY_REGEXP)
```

Then adjust our other calls to the `UnifiedEmbed.register` in the same way.  This starts creating a nice predictable pattern.

Also, not a blocker on this but it would be a great add-on PR.